apps/sdk-tools/src/SdkTools.tsx

```diff
@@ -0,0 +1,224 @@
+  const [staticInput, setStaticInput] = React.useState(
+    '0x000000000000000000000000cb444e90d8198415266c6a2724b7900fb12fc56e000000000000000000000000a0f8904ec48a2775b8a88b40e9c171f05f7d767300000000000000000000000042cedde51198d1773590311e2a340dc06b24cb370000000000000000000000000000000000000000000000004563918244f400000000000000000000000000000000000000000000000000006f30c3f95f913d84000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000140000000000000000000000000000000000000000000000000000000000057e400000000000000000000000000000000000000000000000000000000000000000b051c2da471b8bc8b993fd373b5e6ae8b9aaea1b7a781f0fd41f7f2af7c0bda1',
+  )
+  const [conditionalOrder, setConditionalOrder] = React.useState<ConditionalOrder<unknown, unknown> | undefined>()
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault()
+    console.log('Handler:', handler)
+    console.log('Salt:', salt)
+    console.log('Static Input:', staticInput)
+
+    const conditionalOrder = ordersFactory.fromParams({ handler, salt, staticInput })
+    setConditionalOrder(conditionalOrder)
+    console.log('Conditional Order:', conditionalOrder)
+  }
+
+  return (
+    <Container>
+      <Title>SDK-tools</Title>
+      <Subtitle>Composable CoW</Subtitle>
+      <Address>Address: {COMPOSABLE_COW}</Address>
+
+      <Form onSubmit={handleSubmit}>
+        <Label>
+          Handler:
+          <Input type="text" value={handler} onChange={(e) => setHandler(e.target.value)} />
+        </Label>
+        <Label>
+          Salt:
+          <Input type="text" value={salt} onChange={(e) => setSalt(e.target.value)} />
+        </Label>
+        <Label>
+          Static Input:
+          <Textarea value={staticInput} onChange={(e) => setStaticInput(e.target.value)} />
+        </Label>
+        <Button type="submit">Submit</Button>
+      </Form>
+
+      {conditionalOrder && (
+        <ConditionalOrderContainer>
+          <ConditionalOrderTitle>Conditional Order:</ConditionalOrderTitle>
+          <KeyValueContainer>
+            <KeyValuePair>
+              <Key>Id:</Key>
+              <Value>{conditionalOrder.id}</Value>
+            </KeyValuePair>
+            <KeyValuePair>
+              <Key>Order Type:</Key>
+              <Value>{conditionalOrder.orderType}</Value>
+            </KeyValuePair>
+            <KeyValuePair>
+              <Key>isValid:</Key>
+              <Value>
+                <IsValid isValid={conditionalOrder.isValid()} />
+              </Value>
+            </KeyValuePair>
+            <KeyValuePair>
+              <Key>isSingleOrder:</Key>
+              <Value>
+                <BooleanValue value={conditionalOrder.isSingleOrder} />
+              </Value>
+            </KeyValuePair>
+            {Object.entries(conditionalOrder.data as Object).map(([key, value]) => {
+              if (!isNaN(Number(key))) return null // Ignore numeric keys
```

Comment:
@shoom3301 , I'm not sure our SDK implementation is great. I don't think we should have this numeric values. 

Probably this has to do with how ethersjs gives you an array-lake object that also have the properties, so you can consume it as an object or an array. 

I would think here we just need the object with the data

Reply:
https://github.com/cowprotocol/cow-sdk/pull/226